### PR TITLE
Video banner to image banner

### PR DESCRIPTION
**Pull request by mike-allaway**
Karen originally asked for a video banner but has now decided against it, so this PR converts it to an image banner.

The image is a collage that I made using snippets of letters provided by Karen.

### CODE REVIEW

**Comment by mike-allaway** [comment]
Before:

Screenshot 2021-05-05 at 21 35 00

After:

Screenshot 2021-05-05 at 21 34 53

File: django/core/static/css/custom.css [hunk 1]
```diff
@@ -106,28 +106,32 @@ main a {
...
 /* ---------- Welcome ---------- */
 
-#welcome-videobanner {
-    max-height: 40em;
-    overflow: hidden;
+#welcome-banner {
+    min-height: 75vh;
+    background-image: url('../images/welcome/main-banner.jpg');
+    background-size: cover;
+	background-position: center;
```
Comment (ceryslewis) [inline django/core/static/css/custom.css:113]: Very minor issue of spacing for lines 113 - 115

**Review by ceryslewis — APPROVED** [review]
All looks good other than the very minor issue of spacing mentioned in comment

File: django/core/static/css/custom.css (hunk 1 above)
Comment (mike-allaway) [reply django/core/static/css/custom.css]: thanks for spotting. causes by the dreaded tabs instead of spaces. all sorted now, cheers

**Pull request merged**